construct-binary-tree-from-preorder-and-inorder-traversal/f-exuan21.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Definition for a binary tree node.
+ * public class TreeNode {
+ *     int val;
+ *     TreeNode left;
+ *     TreeNode right;
+ *     TreeNode() {}
+ *     TreeNode(int val) { this.val = val; }
+ *     TreeNode(int val, TreeNode left, TreeNode right) {
+ *         this.val = val;
+ *         this.left = left;
+ *         this.right = right;
+ *     }
+ * }
+ */
+
+ // time : O(n) 
+ // space : O(n)
+ // n은 트리 노드 수
+
+ class Solution {
+
+    private int i = 0;
+    Map<Integer, Integer> map = new HashMap<>();

```

Comment:
nit: 접근 제한자를 일관적으로 붙이거나 생략해주면 불필요한 오해의 소지를 없앨 수 있을 것 같아요.

```suggestion
    private int i = 0;
    private Map<Integer, Integer> map = new HashMap<>();
```